frontend/src/pages/create_campaign/index.tsx

```diff
@@ -169,8 +239,30 @@ const CreateCampaign = () => {
         });
         navigate("/dashboard");
       })
-      .catch(() => {
-        console.error("something went wrong");
+      .catch(async (err) => {
+        if (err instanceof FetchError) {
+          try {
+            const data = (await err.resp.json()) as string;
```

Comment:
should be able to use `const data: string = ` here, the example i showed used `as` to coerce a union type variable to a single type